table-api/src/main/java/io/deephaven/api/TableOperations.java

```diff
@@ -517,6 +519,23 @@ TOPS raj(TABLE rightTable, Collection<? extends JoinMatch> columnsToMatch,
 
     // -------------------------------------------------------------------------------------------
 
+    TOPS updateBy(UpdateByClause operation);
```

Comment:
Should the JavaDoc be lifted up to here, rather than on `Table`?

Reply:
Maybe? At least right now, the javadoc refers to `#getRowSet` which isn't a concept at the TableOperations level. We've been a bit inconsistent in these regards - some of the TableOperations have javadoc, others just on Table.